pkg/universe.dagger.io/python/image.cue

```diff
@@ -0,0 +1,14 @@
+package python
+
+import (
+	"universe.dagger.io/docker"
+)
+
+#Image: {
+	version: *"3.10-alpine" | string
+	_source: "python:\(version)"
+
+	docker.#Pull & {
+		source: _source
```

Comment:
Nit: You could directly interpolate the string, `_source` isn't useful IMO

Reply:
Agreed. Done in the latest push